lib/angular-payments.js

```diff
@@ -467,6 +485,15 @@ angular.module('angularPayments', []);angular.module('angularPayments')
     $target = angular.element(e.currentTarget);
     val = $target.val() + digit;
 
+    // handle the user messing up the separator
```

Comment:
if user have this format: 11/10/18 and it became 11/10-18, will this logic fix it?